### PR TITLE
Update proposal info

### PR DESCRIPTION
**Pull request by zaelgohary**
### Description

image

### Changes

- Edit welcome msg
- Update proposal info

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1757

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by 0oM4R — APPROVED** [review]
image

i think we can do small enhancement

```
    userFarms.value = ( await gridProxyClient.farms.list({
      twinId: profile.value.twinId,
    })).data as unknown as FarmInterface[];
```

File: packages/playground/src/dashboard/dao_view.vue [hunk 1]
```diff
@@ -280,8 +281,11 @@ onMounted(async () => {
     proposals.value = await grid?.dao.get();
     activeProposals.value = proposals.value.active.filter(proposal => proposal.hash);
     inactiveProposals.value = proposals.value.inactive.filter(proposal => proposal.hash);
+    const { data } = await gridProxyClient.farms.list({
```
Comment (amiraabouhadid) [inline packages/playground/src/dashboard/dao_view.vue:284]: you should get only farms with up nodes,  not all user farms when casting vote

**Review by amiraabouhadid — CHANGES_REQUESTED** [review]

**Review by 0oM4R — APPROVED** [review]

**Review by MohamedElmdary — APPROVED** [review]

File: packages/playground/src/dashboard/dao_view.vue [hunk 2]
```diff
@@ -204,6 +204,7 @@
             <div class="textContainer">
               <h2>General</h2>
               <span>A proposal can be created by one of the council members of tfchain.</span>
+              <span> Only farms with one up node or more are allowed to vote.</span>
               <span> Once a proposal has reached it's timelimit, voting stops and a proposal can be closed.</span>
               <span>
                 A proposal is either approved or dissapproved based on the majority of the weights of yes / no votes
```
Comment (amiraabouhadid) [inline packages/playground/src/dashboard/dao_view.vue:210]: "A proposal is either approved or rejected by the weighted votes majority."

**Review by amiraabouhadid — CHANGES_REQUESTED** [review]

File: packages/playground/src/dashboard/dao_view.vue [hunk 3]
```diff
@@ -203,15 +203,21 @@
...
-              <span> Once a proposal has reached it's timelimit, voting stops and a proposal can be closed.</span>
+              <span
+                >TFChain council members have exclusive authority to generate proposals, while owners of farms
+                containing one or more up nodes can cast votes.</span
+              >
+              <span> The voting process concludes at the designated maturity date and time of the proposal.</span>
               <span>
-                A proposal is either approved or dissapproved based on the majority of the weights of yes / no votes
-                (50%).</span
+                Decisions regarding the proposal's acceptance or rejection are determined by a majority of weighted
```
Comment (amiraabouhadid) [inline packages/playground/src/dashboard/dao_view.vue:212]: sounds good, just "the majority"

File: packages/playground/src/dashboard/dao_view.vue [hunk 4]
```diff
@@ -203,15 +203,21 @@
...
+                votes.</span
               >
               <span>
-                A minimal participation threshold must be met. If there are not enough votes and the timelimit is
-                reached, the proposal is dissapproved.</span
+                However, a minimum participation threshold must be attained in order for the voting process to be
+                considered valid.
+              </span>
+              <span>
+                If the vote count is insufficient and the time limit is reached, the proposal will be rejected.</span
```
Comment (amiraabouhadid) [inline packages/playground/src/dashboard/dao_view.vue:220]: "and the time limit has been reached"

Comment (zaelgohary) [reply packages/playground/src/dashboard/dao_view.vue:220]: "limit is reached" is also grammatically correct.

**Review by amiraabouhadid — APPROVED** [review]

**Pull request merged**